backend/src/apis/IotaConnector.ts

```diff
@@ -0,0 +1,103 @@
+import { Client, ClientBuilder } from '@iota/client'
+import { MessageWrapper } from '@iota/client/lib/types'
+
+import { CONFIG } from '@/config'
+import { backendLogger as logger } from '@/server/logger'
+
+// Source: https://refactoring.guru/design-patterns/singleton/typescript/example
+// and ../federation/client/FederationClientFactory.ts
+/**
+ * A Singleton class defines the `getInstance` method that lets clients access
+ * the unique singleton instance.
+ */
+// eslint-disable-next-line @typescript-eslint/no-extraneous-class
+class IotaClientSingleton {
+  private static IotaClient: Client
+
+  /**
+   * The Singleton's constructor should always be private to prevent direct
+   * construction calls with the `new` operator.
+   */
+  // eslint-disable-next-line no-useless-constructor, @typescript-eslint/no-empty-function
+  private constructor() {}
+
+  /**
+   * The static method that controls the access to the singleton instance.
+   *
+   * This implementation let you subclass the Singleton class while keeping
+   * just one instance of each subclass around.
+   */
+  public static getInstance(): Client | null {
+    if (!CONFIG.IOTA || !CONFIG.IOTA_API_URL) {
+      logger.info(`Iota are disabled via config...`)
+      return null
```

Comment:
Gäbe es hierfür nicht eine bessere Lösung als ``return null``?
Dieser return macht dass in der definition Client | null zurück gegeben wird..